make/verify-dependencies.mk

```diff
@@ -0,0 +1,12 @@
+.PHONY: verify-dependencies
+## Runs commands to verify after the updated dependecies of toolchain-common/API(go mod replace), if the repo needs any changes to be made
+verify-dependencies: tidy generate vet build test lint-go-code
+
+.PHONY: tidy
+tidy: 
+	go mod tidy
+
+.PHONY: vet
+vet:
+	go vet ./...
```

Comment:
btw, all these commands are related to go - you could maybe add it to the `go.mk` file instead of creating a new one - to not split the related targets between two files

Reply:
done